recipes-debian/mesa/mesa_debian.bb

```diff
@@ -0,0 +1,35 @@
+# base recipe: meta/recipes-graphics/mesa/mesa_19.0.1.bb
+# base branch: warrior
+# base commit: 3bc700a12d4f21a5c7a1575da0972f9f78b4a091
+
+require ${COREBASE}/meta/recipes-graphics/mesa/mesa.inc
+
+BPN = "mesa"
```

Comment:
You don't have to set BPN in this case.

Reply:
I have removed it from the recipe and fixed a problem.